RELEASE_NOTES.md

```diff
@@ -6,6 +6,15 @@ Release notes added on January 11, 2017
 
 ## Releases
 
+### 2017-04-10 Studentify version 2.0
+
+- Generalized `studentify`:
+  - Add configuration option for test code folders
+    - configuration file `.student-settings.conf` in course master contains a settings line:
+  - `studentify` will now abort if the course master source folder contains uncommited changes
```

Comment:
This is nice. It was confusing before when uncommitted changes wouldn't show up in the student repo after running studentify.